io/plugins/eu.esdihumboldt.hale.io.csv/src/eu/esdihumboldt/hale/io/csv/writer/AbstractTableInstanceWriter.java

```diff
@@ -229,9 +236,48 @@ private void addProperty(List<String> headerRow, Map<String, Object> row, Object
 		if (!headerRow.contains(propertyTypeName)) {
 			headerRow.add(propertyTypeName);
 		}
+
+		if (property instanceof GeometryProperty<?>) {
+			GeometryProperty<?> geometryProperty = (GeometryProperty<?>) property;
+
+			Geometry geometry = geometryProperty.getGeometry();
+			property = simplifyGeometry(geometry);
```

Comment:
Like this the CRS information gets lost, as only the geometry is retained, not the CRS information.

Reply:
True. Then using the simplified geometry and the CRS, I have created a new DefaultGeometryProperty to be used for writing the files.